internal/controller/apiextensions/definition/reconciler.go

```diff
@@ -403,7 +403,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 			"desired-version", desired.APIVersion))
 	}
 
-	ro := CompositeReconcilerOptions(r.options.Features, d, r.client, r.log, r.record)
+	ro := CompositeReconcilerOptions(r.options.Features, d, r.client, r.log, r.record, r.options.PollInterval)
```

Comment:
Perhaps `CompositeReconcilerOptions` should just take `r.options` as an argument?

Reply:
I thought about that but wasn't sure how much to refactor.  I'll make that change.

Reply:
Of course now it has four of the six Reconciler components as arguments, so it almost seems more efficient to just make it 
```
func (r *Reconciler) CompositeReconcilerOptions(d *v1.CompositeResourceDefinition) []composite.ReconcilerOption {
```
but I don't know if that has other undesirable consequences.

Reply:
I don't feel strongly about whether we go with a method or a function here.

Reply:
There is less churn leaving it as a function so I went with that.